cpnest/proposal.py

```diff
@@ -5,10 +5,22 @@
 from abc import ABCMeta,abstractmethod
 import random
 from random import sample,gauss,randrange,uniform
-from scipy.interpolate import LSQUnivariateSpline
-from scipy.signal import savgol_filter
 from scipy.stats import multivariate_normal
 from scipy.special import logsumexp
+from .neighbors import constructKDTree, queryKDTree
```

Comment:
neighbors.py missing from commit

Reply:
sorry, part of other experimental changes, will remove it in the next commit